operator/roles/forkliftcontroller/templates/api/deployment-forklift-api.yml.j2

```diff
@@ -19,6 +19,7 @@ spec:
         app: {{ app_name }}
         service: {{ api_service_name }}
     spec:
+      serviceAccountName: forklift-controller
```

Comment:
it would be better to introduce another, more limited, service account for forklift-api but we can do this separately